core/templates/services/context.service.spec.ts

```diff
@@ -64,6 +64,24 @@ describe('Context service', () => {
       expect(ecs.getQuestionPlayerIsManuallySet()).toEqual(false);
       expect(ecs.isInQuestionPlayerMode()).toEqual(false);
     });
+
+    it('should affirm the page context to be exploration player or editor',
```

Comment:
Duplicate description.

Reply:
This test seems to be duplicated 2 more times.

Reply:
Removed other 2 tests and duplicated description. Done!